src/MSBuild.UnitTests/CommandLineSwitches_Tests.cs

```diff
@@ -1447,9 +1447,11 @@ public void HelpMessagesAreValid()
                     // All lines should be 80 characters or less
                     Assert.True(helpMessageLines[i].Length <= 80, $"Line {i + 1} of '{item.Key}' should be no longer than 80 characters.");
 
+                    string trimmedLine = helpMessageLines[i].Trim();
+
                     if (i == 0)
                     {
-                        if (helpMessageLines[i].Trim().StartsWith("/") || helpMessageLines[i].Trim().StartsWith("@"))
+                        if (trimmedLine.StartsWith("/") || trimmedLine.StartsWith("-") || trimmedLine.StartsWith("@"))
```

Comment:
This unit test only tests our usage text.  I think you should remove the `StartsWith("/")` so that we make sure all usage messages are correct